packages/activity/src/index.ts

```diff
@@ -297,6 +297,8 @@ export class Context {
     if (store === undefined) {
       throw new Error('Activity context not initialized');
     }
+    store.sleep = store.sleep.bind(store);
+    store.heartbeat = store.heartbeat.bind(store);
```

Comment:
Instead of this you could define sleep as:

```ts
sleep: (ms: Duration): Promise<void> => {
}
```

Same goes for heartbeat.

Reply:
I guess they would be equivalent. Though, Current SDK and samples code style heavily prefers `function` methods over `array function` methods.

Reply:
@bergundy Should I change it to arrow function? It's okay but in the future if a normal function using `this` is added, it will break if not bound as in the commit above.

Reply:
@bergundy Done

Reply:
@bergundy Done

Reply:
I also test it using the sdk samples so it's working